_episodes/56.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: episode
+title: "55: The vast possibilities of VAST + Trino"
+date: 2024-02-22
+tags: trino vast database catalog
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 56: The vast possibilities of VAST + Trino"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Colleen Tartow](https://linkedin.com/in/colleen-tartow-phd), Field CTO and
+  Head of Strategy at [VAST Data](https://vastdata.com/).
+* [Roman Zeyde](https://www.linkedin.com/in/roman-zeyde/), Senior Software
+  Engineer at [VAST Data](https://vastdata.com/).
+
+## Release 439
+
+[Trino 439](https://trino.io/docs/current/release/release-439.html)
+
+* New caching layer for Delta Lake, Hive, and Iceberg!
+* Fix for setting session properties on catalogs with a `.` in the name.
+* Fix for reading Snappy data.
+
+## Concept of the episode: The VAST data platform
+
+Part database, part data warehouse, part data lake, describing VAST in one
+sentence is not the easiest undertaking. You can talk about the features like
+deep write buffers with underlying flash columnar storage, the automatic
+contextual layer added on top of the data, or the similarity-based global
+compression that more than makes up for the smaller columnar chunks and makes it
+so much faster to find exactly the data you're looking for.
+
+So what is VAST? It's a state-of-the-art data platform. Why are we talking
+about it on the Trino Community Broadcast? A world-class data storage solution
+still needs a world-class query engine, and its speed paired with Trino's makes
+for a brilliant pair. We're diving into how it works, why it was designed the
+way it was, and maybe talk about the really cool
+[performance comparison](https://vastdata.com/database#performance-comparison)
+they have on their website showcasing Trino as their favorite query engine.
+
```

Comment:
Can we also talk about the two aspects of the Trino VAST connector query a VAST database on the one side.. and the fact that VAST also is a compute cluster.. on which you can install Trino as a supported (and best) query engine..  we will talk about this of course but should also flesh it out a bit in the shownotes

Reply:
I'll be happy to describe our connector - what topics would you prefer to discuss?

Reply:
Yep - there are two aspects here and I can speak to the S3 vs DB perspective, how Trino works with each, and throw out some benchmarks.  

Then we can have @rz-vastdata discuss more about the inner workings of the connector.